docs/conventions/mailers.md

```diff
@@ -0,0 +1,78 @@
+[All Conventions](/docs/conventions.md) / Mailers
+
+# Mailers
+
+We follow standard Rails mailer conventions.
+
+## File Structure
+
+```
+app
+└── mailers
+    ├── support_user_mailer.rb # SupportUserMailer
+    └── user_mailer.rb # UserMailer
+```
+
+## Naming Conventions
+
+To create consistency amongst our mailers, we have introduced some scalable conventions.
+
+> ### The essence of a mailer
+>
+> To develop a naming convention, we must first reflect on how our code is being used, and look for some common patterns that apply to most use cases. In doing so, I’ve realised that most mailers can all be boiled down to these common features:
+>
+> #### Recipient
+>
+> The recipient is the entity that will receive the email. This will normally be a person, but it could also be another app or remote service that receives email inputs. Recipients might be your service users, customers, internal admins, or unauthenticated visitors to your website.
+>
+> ### Resource
+>
+> Emails are usually sent out to provide information about a specific resource or domain concept within the system. In other words, emails are usually sent about something relevant to some thing. An article has been published, a subscription has lapsed, a purchase has been completed. These will often be the models that you’ve defined in your code, but they might also be more abstract concepts (like a password reminder).
```

Comment:
Should we include something here about the need to add an `unsubscribe` button to these types of emails?

Reply:
We only need unsubscribe links for marketing emails. Transactional emails do not need unsubscribe links. 😄